docs/dev/cut-a-new-release.md

```diff
@@ -205,5 +205,7 @@ If you need to release a hotfix for a previously released version (that is: not
 
 ## Upgrading android-components to a new version of Glean
 
-On Android, Mozilla products consume the Glean SDK through its wrapper in [`android-components`](https://github.com/mozilla-mobile/firefox-android/tree/main/android-components/).
+On Android, Mozilla products consume the Glean SDK through its wrapper in [`android-components`][ac].
```

Comment:
Well, that's not exactly true anymore. Only the concept-fetch uploader impl lives there now

Reply:
hmmm true. I'll rephrase that.